backends/arm/runtime/EthosUBackend.cpp

```diff
@@ -10,6 +10,11 @@
  * ethos-u-core-driver for hardware interaction.
  */
 
+// Workaround for runtime/core/portable_type/c10/c10/util/Float16-math.h
```

Comment:
@perheld  have a PR on this here: https://github.com/pytorch/executorch/pull/16221